vacasa/connect/connect.py

```diff
@@ -477,6 +477,70 @@ def get_unit_amenities_reduced(self, params: dict = None):
 
         return self._iterate_pages(url, headers, params)
 
+    def add_unit_amenity(self,
+                         amenity_id: int,
+                         unit_id: int,
+                         amenity_value: int = None,
+                         internal_notes: str = None,
+                         instructions: str = None,
+                         notes: str = None,
+                         provider_id: int = None) -> dict:
+        """Add a new unit amenity in Connect.
+
+        Args:
+            amenity_id: An amenity ID.
+            unit_id: A unit ID.
+            amenity_value: An amenity value.
+            internal_notes: An internal note.
+            instructions: An instruction.
+            notes: A note.
+            provider_id: A provider ID.
+
+        Returns:
+            json response for success
+        """
+        url = f"{self.endpoint}/v1/unit-amenities"
+        headers = self._headers()
+        payload = {
+            'data': {
+                'attributes': {
+                    'amenity_id': amenity_id,
+                    'unit_id': unit_id,
+                    'amenity_value': amenity_value,
+                    'internal_notes': internal_notes,
+                    'instructions': instructions,
+                    'notes': notes,
+                    'provider_id': provider_id,
+                }
+            }
+        }
+
+        r = self._post(url, json=payload, headers=headers)
```

Comment:
in the other connect api endpoints, we just return the `self._post` @engstrom  Do we want to keep the handling of json parsing and other error status on the project calling the SDK?